### PR TITLE
added fireplace in every captain's office, fixed bug with fireplace

### PR DESCRIPTION
**Pull request by ghost**
Images maybe never

### CODE REVIEW

**Comment by ghost** [comment]
Does it still show from all sides of a wall?

**Comment by ghost** [comment]
Nope!

File: code/ATMOSPHERICS/hvac/spaceheater.dm [hunk 1]
```diff
@@ -335,23 +335,30 @@
 	light_range_on = 6
 	nocell = 2
 	density = 0
+	pixel_y = 30
```
Comment (ghost) [inline code/ATMOSPHERICS/hvac/spaceheater.dm:338]: @TermedClepe it uses this to appear on the wall but not be on it, like every other item that's on a wall on one side and not the other.

**Comment by ghost** [comment]
It's gone!

**Comment by ghost** [comment]
AAAAAAAAAA

**Comment by ghost** [comment]
don't panic don't panic don't panic I FIXED IT

**Comment by 9600bauds** [comment]
images when

**Comment by ghost** [comment]
@9600bauds use your imagination

**Comment by 9600bauds** [comment]
wow that looks good

**Comment by ghost** [comment]
imagine this but in defficiency taxi & meta as well
office

**Comment by clusterfack** [comment]
Doesn't mesh at all

**Comment by 9600bauds** [comment]
that looked better in my imagination

**Comment by ghost** [comment]
totally meshes

**Comment by generalveers** [comment]
It meshes, and it's comfy as hell. If you can't build them or order them, at least include one.

**Comment by 9600bauds** [comment]
I would make the mantletop a little bit thicker or more visible, personally

**Comment by SangriaMonkey** [comment]
> wood floors and miniature fireplaces in an industrial space station

**Comment by ghost** [comment]
conflicts are kill

**Comment by ghost** [comment]
I forgot to run that cleanmap thing damn it

**Comment by clusterfack** [comment]
You didnt clean the map

**Comment by ghost** [comment]
I don't agree there is any controversy here. You claim that it fit the setting, but I don't know how carpets or wooden floors fit the setting either. The point of the captain's office in my view is to be a classy area for the most high ranking member of the station, and there's nothing more classy than a fireplace.

**Comment by clusterfack** [comment]
How many people seem to like it here vs how many people did not?

**Comment by PJB3005** [comment]
Erm, a **brick**, wood-fuled **open** fireplace on a **space station**?

**Comment by ghost** [comment]
@PJB3005 so what?

@clusterfack >implying the codebase is representative of the player base

**Comment by ghost** [comment]
On realisticness in the setting: Why do we have a jukebox? Why do we have a fax machine? Why do we have a boombox? 

On the practicalities of fireplaces chimneys: we have toilets, sinks, and showers which lead to no water source, what is the problem with chimneys? 

On the practicalities of brick: this is not a constructible object. that's a non issue.

**Comment by TheEliminator16** [comment]
I'm here to shill my sprite as Celt told me to. I'm not really sure what he meant by that but I'm here, so...

**Comment by ghost** [comment]
remove controversial please, I held a vote on whether or not this should be added:

Vote Result: Yes with 16 votes
     No had 1 vote

**Comment by PJB3005** [comment]
Celt asked very nicely for me to remove [controversial]

**Comment by clusterfack** [comment]
Did you include a picture, and what did your vote actually say, was it one of your famous Celt (tm) votes

**Comment by ghost** [comment]
The vote was, "Should a fireplace be mapped in to the captain's office?"

**Comment by ghost** [comment]
also when looking that up I found out there was some poor guy trying to make his own fireplace but it's not constructable. RIP

**Comment by N3X15** [comment]
2 compiles out of 4 completed and green, the other 2 stalled.  Merging since Travis a shit.

**Comment by ghost** [comment]
no no no

**Comment by ghost** [comment]
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Pull request merged**